dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/QueryFilter.java

```diff
@@ -89,9 +92,10 @@ public String getSqlOperator()
             return null;
         }
 
-        return OPERATOR_MAP.get( operator );
+        return OPERATOR_MAP.get( operator ).apply( filter.contains( NV ) );
```

Comment:
I'm not sure if "filter" can be bull here. Maybe better do something like StringUtils.trimToEmpty()

Reply:
it shouldn't, however I'll add a check just in case

Reply:
done